CharityDB/1 tables.sql

```diff
@@ -0,0 +1,17 @@
+use CharityDB
+go
+drop table if exists Donor
+go 
+
+create table dbo.Donor(
+    DonorId int not null identity primary key,
+    FirstName varchar(25) not null constraint ck_Donor_FirstName_cannot_be_blank check(FirstName <> ''),
+    LastName varchar(25) not null constraint ck_Donor_LastName_cannot_be_blank check(LastName <> ''),
```

Comment:
To prevent future mistakes, what about making the name and address unique?

Reply:
I would make the address and postcode together unique, as it is possible to have the same road name in different areas.

Reply:
see comment below.